ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -188,11 +203,15 @@
                   class="submitting-party"
                 >
                   <ContactInformation
-                    :contactInfo="getMhrTransferSubmittingParty"
+                    :contactInfo="isChangeLocationActive
+                      ? getMhrTransportPermit.submittingParty
+                      : getMhrTransferSubmittingParty"
```

Comment:
@dimak1 Issue is here sir 👍

Reply:
See below, this may be ok, could just be the setter?

Reply:
Yea, thank you!